docs/src/main/asciidoc/load-balancer.adoc

```diff
@@ -0,0 +1,28 @@
+== LoadBalancer for Kubernetes
```

Comment:
Thanks for adding the docs. Please make sure you include this file [here](https://raw.githubusercontent.com/spring-cloud/spring-cloud-kubernetes/master/docs/src/main/asciidoc/spring-cloud-kubernetes.adoc).

Reply:
Done